frontend/src/app/testResults/uploads/__snapshots__/CsvSchemaDocumentation.test.tsx.snap

```diff
@@ -1610,98 +1760,88 @@ exports[`CsvSchemaDocumentation tests CsvSchemaDocumentaton matches snapshot 1`]
               <div
                 class="grid-col-auto"
               >
-                patient_preferred_language
+                <code>
+                  patient_preferred_language
+                </code>
               </div>
             </div>
             <div
               class="grid-row margin-bottom-05 border-base-lighter border-top-1px padding-top-1"
+              data-testid="description"
             >
               <div
                 class="grid-col-4 text-base"
               >
-                Value type
+                Description
               </div>
               <div
-                class="grid-col-auto"
+                class="grid-col-8"
               >
-                Text
+                <div>
+                  Look up a Language Concept Code or Name from the  
+                  <a
+                    class="usa-link"
+                    href="https://phinvads.cdc.gov/vads/ViewValueSet.action?id=D0858308-9AB3-EA11-818F-005056ABE2F0#"
+                    rel="noreferrer noopener"
+                    target="_blank"
+                  >
+                    ISO-639 table in the PHIN VADS system
+                  </a>
+                </div>
               </div>
             </div>
             <div
               class="grid-row margin-bottom-05 border-base-lighter border-top-1px padding-top-1"
+              data-testid="examples"
             >
               <div
                 class="grid-col-4 text-base"
               >
-                Example(s)
+                Example
+                s
               </div>
               <ul
                 class="grid-col-8 prime-ul margin-top-0"
               >
-                <li>
-                  <code>
-                    eng
-                  </code>
-                   or 
-                  <code>
-                    English
-                  </code>
-                </li>
-                <li>
-                  <code>
-                    spa
-                  </code>
-                   or 
-                  <code>
-                    Spanish
-                  </code>
-                </li>
-                <li>
-                  <code>
-                    fre
-                  </code>
-                   or 
-                  <code>
-                    French
-                  </code>
-                </li>
-                <li>
-                  <code>
-                    jpn
-                  </code>
-                   or 
-                  <code>
-                    Japanese
-                  </code>
+                <li
+                  class="bullet-list"
+                >
+                  <em>
+                    eng or English
+                  </em>
+                </li>
+                <li
+                  class="bullet-list"
+                >
+                  <em>
+                    spa or Spanish
+                  </em>
                 </li>
```

Comment:
```suggestion
                <li
                  class="bullet-list"
                >
                  <em>
                    eng
                  </em>
                   or <em>English</em>
                </li>
                <li
                  class="bullet-list"
                >
                  <em>
                    spa
                  </em>
                   or <em>Spanish</em>
                </li>
```

Reply:
Made one last tiny edit. Otherwise LGTM!

Reply:
Thanks, I'll merge this as it and ill include this change in a separate PR